feeluown/app.py

```diff
@@ -169,6 +169,7 @@ def attach_attrs(app):
 
     if app.mode & app.DaemonMode:
         app.server = FuoServer(app)
+        app.ws_server = FuoWebsocketServer(app)
```

Comment:
Perhaps make the WebSocket server configurable. We are starting too many servers.